R/densityClust.R

```diff
@@ -279,6 +328,66 @@ plotMDS.densityCluster <- function(x, ...) {
         legend('topright', legend=c('core', 'halo'), pch=c(19, 1), horiz=TRUE)
     }
 }
+#' Plot observations using t-distributed neighbor embedding and colour by cluster
+#' 
+#' This function produces an tSNE scatterplot based on the distance matrix of the
+#' densityCluster object (if there is only the coordinates information, a distance
+#' matrix will be calculate firstly), and, if clusters are defined, colours each 
+#' observation according to cluster affiliation. Observations belonging to a cluster
+#' core is plotted with filled circles and observations belonging to the halo with
+#' hollow circles. 
+#' 
+#' @param x A densityCluster object as produced by \code{\link{densityClust}}
+#' 
+#' @param ... Additional parameters. Currently ignored
+#' 
+#' @examples
+#' irisDist <- dist(iris[,1:4])
+#' irisClust <- densityClust(irisDist, gaussian=TRUE)
+#' plot(irisClust) # Inspect clustering attributes to define thresholds
+#' 
+#' irisClust <- findClusters(irisClust, rho=2, delta=2)
+#' plotTSNE(irisClust)
+#' split(iris[,5], irisClust$clusters)
+#' 
+#' @seealso \code{\link{densityClust}}
+#' 
+#' @export
+#' 
+plotTSNE <- function (x, ...) {
+    UseMethod("plotTSNE", x)
```

Comment:
No need for the x - UseMethod('plotTSNE') is sufficient

Reply:
fixed